capi-matrix/README.md

```diff
@@ -0,0 +1,50 @@
+# RFC 0015 - CAPI matrix per provider
+
+## Introduction
+
+We have a cluster-<provider> chart for every provider that we support. It has a set of templates that allow our customers to create a cluster app with all necessary resources required by a Cluster API implementation. 
+
+In the values of these charts we define the Kubernetes version of the cluster and the image that machine will use to run. These images contain the Kubernetes components with a specific version. 
+
+One of the goal in this RFC is decouple Kubernetes version from image names so customer does not need to know which image correspond to which version. At the same time we might want to provide new OS version when new CVE appears without necessarily change the Kubernetes version. Finally having a list of kubernetes version certified as ready to be used benefit to disallow users to create cluster in flawy versions.
+
+## User stories
+
+- As a customer, I want to not worry about the kubernetes version and rely on Giant Swarm default version provided.
+
+- As Giant Swarm staff, I don't want to expend time upgrading cluster manually.
+
+- As Giant Swarm staff, I would like to forbid a specific Kubernetes/OS version to be used in case of security constraints.
+
+- As customer, I want to have the option to have a cluster pinned to a version.
```

Comment:
Can we include reasons here? 

And what boundaries? 
Will it be pin to a specific kubernetes/OS combo? Pinned to a patch? Pinned to a major?
Will we let them pin for 6 months? 1 year?

Reply:
sure